game/game.py

```diff
@@ -24,12 +25,12 @@ def main():
         print_report(winner, score, user_choice, comp_choice)
 
 
-def get_made_choices(user_choice, comp_choice, choices):
+def get_made_choices(user_choice: str, comp_choice: str, choices: dict) -> Any:
```

Comment:
choices should be Dict, check examples in google
return you can change code to be in tuple, and hint typing will be Tuple[str, str]